pyscriptjs/tests/integration/test_02_display.py

```diff
@@ -153,7 +152,10 @@ def display_hello():
         text = self.page.locator("id=second-pyscript-tag").inner_text()
         assert text == "hello"
 
-    @skip_worker("FIXME: display()")
+    @pytest.mark.skip(
+        "FIXME: on CHrome fails with the error:"
+        ' The interpreter "py" was not found. Available interpreters are: "py-script", "pyodide".'
```

Comment:
I think all these warnings are misleading ... the project works as expected and as designed but all tests need to be changed ... you can try this locally and see this very specific test just works:

```html
<!doctype html>
<script type="module" src="https://cdn.jsdelivr.net/npm/@pyscript/core"></script>
<py-script>
  def display_hello(event):
      from pyscript import display
      display('hello', target=event.target.id)
</py-script>
<button id="my-button" py-click="display_hello">Click me</button>
```

Most notably:

  * we had a very lengthy discussion around not having brackets and code evaluation in listeners ... no `python_method()` is allowed anymore, `python_method` *must* be used instead
  * every method receives the `event` that was triggered
  * the `display` is not a global leak anymore, you need to `from pyscript import display` in every single display test

Reply:
Honestly, that's the goal of this PR and resuming these tests. Not to say what's right or wrong but to have a clear map of what differs and decide, case by case, what should be the behavior and either declare the old version deprecated to use the new (hopefully with a deprecation warning) or fix because the current behavior isn't what we expect. 

The intent is to have visibility

Reply:
There is no code evaluation in events anymore so it can't be deprecated, it will just break. We can deprecate everything we want in PyScript "classic" though and release latest classic as a patch with all deprecations and changes to help migrating to next. I think this might be a great approach!